app/src/main/java/com/ucmobiledevelopment/freeways/MainViewModel.kt

```diff
@@ -13,14 +13,13 @@ import com.ucmobiledevelopment.freeways.service.IncidentService
 import kotlinx.coroutines.launch
 
 class MainViewModel(var incidentService : IIncidentService = IncidentService()) : ViewModel(){
-    internal val NEW_INCIDENT = "New Incident"
+    private val NEW_INCIDENT = "New Incident"
     var incidents : MutableLiveData<List<Incident>> = MutableLiveData<List<Incident>>()
     var user : User? = null
 
-    private lateinit var firestore: FirebaseFirestore
+    private var firestore: FirebaseFirestore = FirebaseFirestore.getInstance()
 
     init{
-        firestore = FirebaseFirestore.getInstance()
```

Comment:
lateinit seemed unnecessary, so I removed the lateinit modifier and got the instance as the var was declared.

Reply:
Either way works.  I'd probably go with what the reviewer recommends here.